acceptance/tests/config/apply_file_metadata_specified_in_config.rb

```diff
@@ -1,5 +1,8 @@
 test_name "#17371 file metadata specified in puppet.conf needs to be applied"
```

Comment:
the ticket # here is a bit confusing.  should we remove or link to (and create) a testrail entry?